wcfsetup/install/files/lib/system/cache/eager/data/LanguageCacheData.class.php

```diff
@@ -62,6 +60,68 @@ public function getLanguageCategory(string $categoryName): ?LanguageCategory
      */
     public function getLanguageCategoryByID(int $languageCategoryID): ?LanguageCategory
     {
-        return $this->categories[$this->categoryIDs[$languageCategoryID] ?? null] ?? null;
+        $categoryName = $this->categoryIDs[$languageCategoryID] ?? null;
+        if ($categoryName === null) {
+            return null;
+        }
+
+        return $this->getLanguageCategory($categoryName);
+    }
+
+    /**
+     * Returns `true` if the language category with the given category name exists.
+     */
```

Comment:
```suggestion
```

This comment is redundant because this is literally what the name of the method and its return type implies.